### PR TITLE
Expand test coverage

### PR DESCRIPTION
**Pull request by eizamaliev**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
> # ModelArg('instance-segmentation-security-0091')
is there are any issues with this model?

**Comment by eizamaliev** [comment]
> ># ModelArg('instance-segmentation-security-0091')
> is there are any issues with this model?
@vladimir-dudnik no issues, except long execution time. If it ok, I'll uncomment it. Or could add comment

**Comment by vladimir-dudnik** [comment]
> > > # ModelArg('instance-segmentation-security-0091')
> > 
> > 
> > is there are any issues with this model?
> 
> @vladimir-dudnik no issues, except long execution time. If it ok, I'll uncomment it. Or could add comment

ok, would be nice to mention this in code comment, to make it clear why model is turned off from testing. no need to uncomment this, it will not bring any new info as we have internal validation for it

File: demos/formula_recognition_demo/python/utils.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def create_renderer():
     of latex formula could be performed
     """
     command = subprocess.run(["pdflatex", "--version"], stdout=subprocess.DEVNULL,
-                             stderr=subprocess.DEVNULL, check=False)
+                             stderr=subprocess.DEVNULL, check=False, shell=True)
```
Comment (Wovchena) [inline demos/formula_recognition_demo/python/utils.py:89]: What is this for? My assumption was it is to address @vladimir-dudnik's issue of running on Windows, but I've rechecked and the demo works with `shell=False`.

Comment (eizamaliev) [reply demos/formula_recognition_demo/python/utils.py:89]: @Wovchena I've got a problem running demo on Windows without Latex package installed.

File: demos/tests/cases.py [hunk 2]
```diff
@@ -906,6 +959,21 @@ def single_option_cases(key, *args):
...
     )),
+
+    PythonDemo(name='whiteboard_inpainting_demo', device_keys=['-d'], test_cases=combine_cases(
+        TestCase(options={'-i': TestDataArg('msasl/global_crops/_nz_sivss20/clip_0017/img_%05d.jpg'),
+                          **MONITORS,
+                          '--no_show': None}),
+        [
+            *single_option_cases('-m_i',
+                ModelArg('instance-segmentation-security-0002'),
+                # ModelArg('instance-segmentation-security-0091'),
```
Comment (eizamaliev) [inline demos/tests/cases.py:970]:
```suggestion
                # ModelArg('instance-segmentation-security-0091'), # Slow model
```

File: demos/formula_recognition_demo/python/utils.py (hunk 1 above)
Comment (Wovchena) [reply demos/formula_recognition_demo/python/utils.py:89]: Ok, please explain that in code comment

Comment (eizamaliev) [inline demos/formula_recognition_demo/python/utils.py:89]:
```suggestion
                             stderr=subprocess.DEVNULL, check=False, shell=True) # shell=True is used to process case with no pdflatex installed on Windows
```

Comment (eizamaliev) [reply demos/formula_recognition_demo/python/utils.py]: @Wovchena done

**Comment by Wovchena** [comment]
There is a conflict now, please resolve

**Comment by vladimir-dudnik** [comment]
@eizamaliev bump?

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**